generate-pretty-manifest.js

```diff
@@ -1,43 +1,13 @@
-const { getMostRecentManifest, writeFilePretty } = require('./helpers.js');
+const { getMostRecentManifest, writeFile } = require('./helpers.js');
 
 const mostRecentManifestLoaded = require(`./${getMostRecentManifest()}`);
 
-const tables = [
-  'Activity',
-  'ActivityMode',
-  'ActivityModifier',
-  'ActivityType',
-  'Class',
-  'Collectible',
-  'Destination',
-  'Faction',
-  'Gender',
-  'InventoryBucket',
-  'InventoryItem',
-  'ItemCategory',
-  'ItemTierType',
-  'Milestone',
-  'Objective',
-  'Place',
-  'PlugSet',
-  'PresentationNode',
-  'Progression',
-  'Race',
-  'Record',
-  'SandboxPerk',
-  'SocketCategory',
-  'SocketType',
-  'Stat',
-  'TalentGrid',
-  'Vendor',
-  'VendorGroup'
-];
-
-for (var i = 0; i < tables.length; i++) {
-  writeFilePretty(
-    `./manifest_tables/${tables[i]}.json`,
-    mostRecentManifestLoaded[`Destiny${tables[i]}Definition`]
+Object.entries(mostRecentManifestLoaded).forEach(function([table, tableData]) {
+  table = table.replace('Destiny', '').replace('Definition', '');
```

Comment:
give this var a unique name, instead of replacing the value of `table`, and you can skip reassembly on line 9

also a stricter/expliciter/carefuler way to do this extraction might be something like
`const shortname = table.match(/^Destiny(\w+)Definition$/)[1]`